src/test/java/com/stripe/net/StripeResponseTest.java

```diff
@@ -74,6 +74,6 @@ public void testGetIdempotencyKey() {
   public void testRequestId() {
     final Map<String, List<String>> headerMap = generateHeaderMap();
     final StripeResponse stripeResponse = new StripeResponse(200, chargeBody, headerMap);
-    assertEquals(stripeResponse.requestId(), "req_12345");
+    assertEquals("req_12345", stripeResponse.requestId());
```

Comment:
Wow, I love that there's a check rule for this given how common of a mistake this is.